docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.3"
+services:
+  app:
+    build: .
+    command: pm2-dev ./build/server.js
```

Comment:
This command is executed when starting the container, but even though the pm2 restarts the server, docker doesn't trigger new tsc build. Look into optimising Dockerfile